src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -448,6 +457,31 @@ private function formatRouterConfig(array $config): string
         return trim($configAsString);
     }
 
+    private function formatControllerLink($controller, string $anchorText): string
+    {
+        try {
+            if (\is_array($controller)) {
+                $r = new \ReflectionMethod($controller);
+            } elseif ($controller instanceof \Closure) {
+                $r = new \ReflectionFunction($controller);
+            } elseif (method_exists($controller, '__invoke')) {
+                $r = new \ReflectionMethod($controller, '__invoke');
+            } elseif (!\is_string($controller)) {
+                return $anchorText;
+            } elseif (false !== strpos($controller, '::')) {
+                $r = new \ReflectionMethod($controller);
+            } else {
+                $r = new \ReflectionFunction($controller);
+            }
+        } catch (\ReflectionException $e) {
+            return $anchorText;
+        }
+
+        $fileLink = $this->fileLinkFormatter->format($r->getFileName(), $r->getStartLine());
```

Comment:
`$this->fileLinkFormatter` can be null

Reply:
thanks @chalasr for your feedback, I updated.